homeassistant/components/ecobee/climate.py

```diff
@@ -563,8 +585,49 @@ def extra_state_attributes(self) -> dict[str, Any] | None:
             ),
             "equipment_running": status,
             "fan_min_on_time": self.settings["fanMinOnTime"],
+            "available_sensors": self.remote_sensor_devices,
+            "active_sensors": self.active_sensor_devices_in_preset_mode,
         }
 
+    @property
+    def remote_sensors(self) -> list:
```

Comment:
I think we should [exclude this property as well as `remote_sensor_devices` from recording](https://developers.home-assistant.io/docs/core/entity#excluding-state-attributes-from-recorder-history) by adding them to `_unrecorded_attributes`.